src/components/core/DpDataTable/DpDataTable.vue

```diff
@@ -2,6 +2,7 @@
   <div>
     <table
       ref="tableEl"
+      :aria-label="tableDescription"
```

Comment:
i am not sure if aria-label is sufficient here. Its ment be used for short texts.
Why don't You use 
`<caption class="visibility--hidden" v-text="tableDescription" />`
?

Reply:
If the element has the CSS property `visibility: hidden`, it will not be visible for the screen readers, do we want it?

Reply:
https://webaim.org/techniques/css/invisiblecontent/

Reply:
The table caption can be defined via [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) or [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label). --> https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/table_role

Reply:
sorry. I took the wrong class.  we have a special class for that `hide-visually` (DemosPlanCoreBundle/Resources/client/scss/utility/_display-visibility.scss:91)

Reply:
ah, got it, thx!